test-poc/framework/src/main/java/org/keycloak/test/framework/injection/Registry.java

```diff
@@ -83,6 +84,11 @@ public <T> T getDependency(Class<T> typeClass, InstanceWrapper dependent) {
     }
 
     public void beforeAll(Class testClass) {
```

Comment:
This method no longer has any implementation, perhaps it can be removed?

Reply:
Left it in case we needed it in the future, but better to remove now as it's easy to re-introduce if needed down the line. Removed now.